index.html

```diff
@@ -240,18 +252,27 @@ <h2>Skills</h2>
           </li>
         </ul>
       </section>
-      <section class="get-in-touch">
-        <h3 class="intro-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <section class="contact-us get-in-touch">
+        
+      <div class="intro-text"> <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3></div> 
 <form action="https://formspree.io/f/mpzblelz" method="post">
 
 <ul>
+<li class="desktop-view-input">
+
+    <input type="text" required placeholder="First Name" maxlength="30" name="user_name">
+    
+    
+      <input type="text" class="second-name-input" required placeholder="Last Name" maxlength="30" name="user_name">
+      
+      
 
-<li>
-<input type="text" required placeholder="First Name" maxlength="30" name="user_name">
 </li>
 <li>
   <input required type="email" placeholder="Email address" name="user_email" >
 </li>
+
+
 <li>
   <textarea name="user_message" maxlength="500" required id="message" cols="30" rows="10">Enter text here</textarea>
 </li>
```

Comment:
- [ ] Please fix the font size and font weight for the text beside the contact form in both desktop version and mobile version and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/159787425-a567825d-315c-4911-9d27-47b26eaef126.png) | ![image](https://user-images.githubusercontent.com/68796232/159787310-b431d52a-20f5-42d3-9985-80359ab1c724.png)  |

![image](https://user-images.githubusercontent.com/68796232/159787953-17821506-215b-450f-8287-7a3258579bb7.png)

Reply:
font-weight is the same in both mobile and desktop design, however i will change the font-size for the desktop version to 30px.